### ISSUE TITLE
Reproducing results reported by paper

### ISSUE THREAD

**Issue by arvkr** [mkocabas/pose-residual-network#27]
Given that the full network & training flow is not released by the authors, did anyone actually fully succeed in reproducing the results written in the paper (both the accuracy & speed of 23 FPS)? Either DL framework is ok. Thank you.

**Comment by big-xiao** [comment]
https://github.com/LiMeng95/MultiPoseNet.pytorch,this is website that not fully succeed in reproducing the results.
